src/components/mapping/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { ComponentMeta, ComponentStory, ComponentStoryObj } from "@storybook/react"
+import { Map } from "react-map-gl"
+import { mapboxStyleLight, mapboxStyleDark } from ".."
+import { useColorScheme } from "../../../contexts/ColorSchemeProvider"
+import { Text } from "../../typography/Text/Text"
+import { MapTooltip as Tooltip } from "./Tooltip"
+
+const mapboxAccessToken = import.meta.env.VITE_MAPBOX_TOKEN
+
+export default {
+  title: "Mapping/Tooltip",
+  component: Tooltip,
+  render: (args) => <Template {...args} />,
+} satisfies ComponentMeta<typeof Tooltip>
+
+type Story = ComponentStoryObj<typeof Tooltip>
+
+const Template: ComponentStory<typeof Tooltip> = (args): JSX.Element => {
+  const { colorScheme } = useColorScheme()
+  return (
+    <Map
+      style={{
+        height: "80vh",
+      }}
+      id="uiMap"
```

Comment:
Does this ID do something? Is it used in the apps? If it's not important but still needs to be provided I think we should make the casing more familiar:

```suggestion
      id="ui-map"
```